README.md

```diff
@@ -51,3 +51,46 @@ To run the acf interface with an ACF with a non-default ip address:
 ```bash
 ros2 run ferrobotics_acf acf.py --ros-args -p ip:=192.168.123.132
 ```
+
+### Parameters
+
+The `acf.py` executable exposes the following parameters:
+
+- `ip` -> the acf ip address
+- `f_max` -> the maximum allowed force [N]
+- `initial_force` -> the force generated when no contact is detected [N]
+- `ramp_duration` -> the ramp time of the force once contact is detected [s]
+- `payload` -> the mass attached to the acf [kg]
+- `frequency` -> the polling frequency [Hz]
+- `joint_name` -> the ACF joint name
+
+### Polling mode
+
+If the `frequency` parameter is set, commands will be set to the ACF at that rate. Received force commands will not immediately trigger sending of commands to the ACF in this mode. ACF telemetry will be received at the same rate.
+
+If the parameter is set to zero, commands will only be sent to the ACF when a force command is received. Telemetry is only sent after each force command.
+
+### Joint State Publisher
+
+This node will also act as a joint state publisher when the `joint_name` parameter is set. This may be useful in conjunction with a robot state publisher.
+
+### Services
+
+The `ACF` node creates the following services:
+
+- `/ACF/set_payload` (SetFloat) -> used to set the current payload [kg]
+- `/ACF/set_f_zero` (SetFloat) -> used to set force generated when no contact is detected [N]
+- `/ACF/set_t_ramp` (SetDuration) -> used to set the ramp time of the force once contact is detected [s]
+
+### Publishers
+
+The `ACF` node publishes on the following topic:
+
+- `/ACF/telem` (ACFTelem) -> Contains return information from the ACF. Only published after receiving a force command.
+- `/joint_state` (JointState) -> Publishes the current position of the ACF as a joint state. Only published if the `joint_name` parameter is set.
+
+### Subscribers
+
+The `ACF` node subscribes to the following topic:
+
+- `/ACF/force` (Float32) -> Used to set the current output force of the ACF in Newtons. Telemetry information is sent after receiving a force command.
```

Comment:
Should we be using a `geometry_msgs/Wrench` here instead?

Semantic value of a `float` is really low. Using more specific messages would be better.

Reply:
Opened issue for this, will address in future PR